sdk/scheduler/src/test/java/com/mesosphere/sdk/api/EndpointUtilsTest.java

```diff
@@ -0,0 +1,64 @@
+package com.mesosphere.sdk.api;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.apache.mesos.Protos.Label;
+import org.junit.Test;
+
+import com.mesosphere.sdk.api.EndpointUtils.VipInfo;
+
+/**
+ * Tests for {@link EndpointUtils}.
+ */
+public class EndpointUtilsTest {
+
+    @Test
+    public void testToEndpoint() {
+        assertEquals("foo:5", EndpointUtils.toEndpoint("foo", 5));
+    }
+
+    @Test
+    public void testToMesosDnsEndpoint() {
+        assertEquals("task.svc.mesos:5", EndpointUtils.toMesosDnsEndpoint("svc", "task", 5));
+        assertEquals("task.svc-to-path.mesos:5", EndpointUtils.toMesosDnsEndpoint("/path/to/svc", "task", 5));
```

Comment:
Aside: What is the behavior if the service name they provide is `a.special.service`?